funnel/assets/js/app.js

```diff
@@ -0,0 +1,37 @@
+import { Utils, ScrollActiveMenu, LazyloadImg } from './util';
+
+$(() => {
+  window.HasGeek = {};
+
+  Utils.collapse();
+  Utils.smoothScroll();
+
+  let intersectionObserverComponents = function() {
+    ScrollActiveMenu.init('page-navbar', 'sub-navbar__item', 'sub-navbar__item--active');
+    LazyloadImg.init('js-lazyload-img');
+  };
+
+  if(document.querySelector('#page-navbar') || document.querySelector('.js-lazyload-img')) {
+    if (!('IntersectionObserver' in global &&
+    'IntersectionObserverEntry' in global &&
+    'intersectionRatio' in IntersectionObserverEntry.prototype)) {
+      let polyfill = document.createElement('script');
+      polyfill.setAttribute('type','text/javascript');
+      polyfill.setAttribute('src','https://cdn.polyfill.io/v2/polyfill.min.js?features=IntersectionObserver');
+      polyfill.onload = function() {
+        intersectionObserverComponents();
+      };
+      document.head.appendChild(polyfill);
+    }
+    else {
+      intersectionObserverComponents();
+    }
+  }
+
+  // Send click events to Google analytics
```

Comment:
If swiping moves to a different content area, shouldn't that also be logged to GA?

Reply:
We are sending these events to GA.
On swipe, an click event to prev/next sibling of active nav item(anchor tag) is triggered. Every click on a button or anchor tags are currently send to GA.